src/Map.jsx

```diff
@@ -249,10 +249,12 @@ class Map extends Component {
             marker => this.getStopInfo(route, marker.object.position), this.state.selectedStops,
           )
           : null,
-        getRoutesLayer(geojson),
         ...getVehicleMarkersLayer(route, info => this.displayVehicleInfo(info)),
       ], []);
 
+    const routesLayer = getRoutesLayer(geojson);
```

Comment:
geojson is used for all of the routes we care about, so we can pull it out of the reduce so it is only called once per render.

Reply:
`routeLayer` and `routesLayer` look easy to confuse for one another...thoughts on doing `routeLayers.push(getRoutesLayer(geojson))` instead?